### PR TITLE
Services Page

### PR DESCRIPTION
**Pull request by LiviaFResende**
## Rationale

This if to Update the Services on the First Page
-- Relevant change only in file ServiceList.jsx

## Advice for Reviewers & Testing Notes

- Explain how your change should be reviewed, and where reviewers should direct their attention.

-- I changed the Services Page, title and description only.
-- Relevant change only in file ServiceList.jsx

- Explain (where relevant) how your change can be tested.

-- no tests at this stage yet. Only changes in texts.

## Task Name

- Issue Title

-- Services Page Issue 

## Linting Checklist
- [ ] No commented code
- [ ] Code Formatted nicely (Prettier)
- [ ] PR your own code before you assign reviewers

### CODE REVIEW

**Review by davidtaing — COMMENTED** [review]
Hi @LiviaFResende,
Thank you for sending in the PR.

I'm having a little trouble running the project and all I see is a blank page. Can you please look into this again?

VS Code will tell you if you got any errors in the files you have opened.

If you can double click the error, VS Code will take you to the line.

The window will look like this
Screen Shot 2022-10-14 at 6 30 04 am

To get this window open, you can press ``ctrl + ` `` or go to the menus `View` -> `Terminal`. And then click on the `Problems` tab on the left.

**Comment by davidtaing** [comment]
After you fix the errors, you can make another commit and push it again. The new commit will come here. 

As always let me know if you need any help 👍.

**Comment by LiviaFResende** [comment]
Hi David, 
Could you please check again? I tried to fix the errors. It appears clean from errors here: 

error 1

**Comment by LiviaFResende** [comment]
Hi David, I just sent a new update tidying the words.

File: src/elements/service/ServiceList.jsx [hunk 1]
```diff
@@ -26,11 +26,12 @@ const ServiceList = [
...
         description: 'Check my Social Media for updates.'
-    },
+    }
+    // ,
     // { 
     //     icon: <FiMonitor />,
-    //     title: 'Mobile App Development',
-    //     description: 'I throw myself down among the tall grass by the stream as I lie close to the earth.'
+    //     title: '',
+    //     description: ''
```
Comment (davidtaing) [inline src/elements/service/ServiceList.jsx:34]: It's always a good idea to remove comments like these when we are not using the code.

I like to use comments to help explain what is going on with the code. But you have to keep in mind that your code already tells a story; so you might not need a comment. Most of the time you are not going to need one.

For example:
```ts
  // this creates a dog <-- does not add new information
  function createDog() {
    return new Dog();
  }
```

**Review by davidtaing — COMMENTED** [review]
This one looks good to me. Good job!

File: src/elements/service/ServiceList.jsx (hunk 1 above)
Comment (davidtaing) [reply src/elements/service/ServiceList.jsx:34]: Sorry, I forgot to add that I think we should come back to this one later.

Let's bookmark this with a new GitHub issue for now. 🙂

**Comment by davidtaing** [comment]
Approved! Thank you for your contribution @LiviaFResende!

Just wanted to point out that the boxes are different sizes though. Don't worry, I know this is from the template, let's bookmark this one for later!

image

**Comment by LiviaFResende** [comment]
Thanks David.

All noted. Thank you for spending the time to give us a complete feedback.
It is very good!

Livia Resende

**Comment by davidtaing** [comment]
Opps, looks like merging @kathryn008's changes removed your changes.

I'll fix this up on my side.

**Comment by davidtaing** [comment]
Sorry, it looks good now. Not sure what happened haha.

**Pull request merged**